tripleo-ciscoaci/deployment/opflex/opflex-agent-container-puppet.yaml

```diff
@@ -148,6 +148,50 @@ parameters:
        The starting backoff value for retrying of policy requests, in seconds. The
        backoff is exponential, starting at this value, and continuing up to 16 retries.
     type: number
+  OpflexStatisticsMode:
+    default: 'real'
+    type: string
+  OpflexEnabledStatisticsInterface:
```

Comment:
Can we change the names to be similar to the names opflex agent? For example, use OpflexStatisticsInterfaceEnabled, and OpflexStatisticsInterfaceInterval.